### PR TITLE
Feature/74 update registration page

### PR DESCRIPTION
**Pull request by Hexuan-Z**
Changes
---
On registration page:
- Remove "UVIC ID" / V number field
- Add drop down menu to choose between three faculties: CSC, SENG and ECE

Testing
---
Manual testing

Linked Issue
---
https://github.com/SENG499-Company-4/Frontend/issues/74

Additional Info
---

### CODE REVIEW

File: app/src/components/pages/Register.tsx [hunk 1]
```diff
@@ -31,6 +36,12 @@ function Register() {
       : setHasErrors(true);
   }, [formData.password, formData.confirmPassword, formData.vnum.length]);
```
Comment (sam-warren) [inline app/src/components/pages/Register.tsx:37]: Need to update this dependency array to remove vnum and add faculty.

File: app/src/components/pages/Register.tsx [hunk 2]
```diff
@@ -3,6 +3,11 @@ import Button from '@mui/material/Button';
 import TextField from '@mui/material/TextField';
 import Box from '@mui/material/Box';
 import { Grid, Typography } from '@mui/material';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import { Faculty } from 'components/shared/constants/timetable.constants';
 
 interface IRegisterForm {
```
Comment (sam-warren) [inline app/src/components/pages/Register.tsx:12]: Update registration form interface, replace vnum with faculty, same goes for the formData object on line 21 - we need to remove vnum from the component and replace it with the faculty dropdown value.

**Review by sam-warren — CHANGES_REQUESTED** [review]
Looks good - just a few programmatic changes

**Review by Ahnaf-Ahmed — CHANGES_REQUESTED** [review]
Some small changes along with Sam's and I think you're good to go,

**Review by sam-warren — APPROVED** [review]

**Review by Ahnaf-Ahmed — APPROVED** [review]
Looks great now!

**Pull request merged**